src/makeRobotAccountant.js

```diff
@@ -6,7 +6,28 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  const MAX_CALLS = 3;
+  let callsCount = 0;
+  let isEven = false;
+  const robotAccountant = (x = 0) => {
+    isEven = !isEven;
+
+    if (callsCount <= MAX_CALLS) {
+      callsCount++;
+    }
```

Comment:
You should check even callsCount or not inside second function
Also this check is useless here
```suggestion
    callsCount++;
```